src/nvim/eval.c

```diff
@@ -10383,10 +10383,13 @@ void script_host_eval(char *name, typval_T *argvars, typval_T *rettv)
 
   list_T *args = tv_list_alloc(1);
   tv_list_append_string(args, (const char *)argvars[0].vval.v_string, -1);
-  *rettv = eval_call_provider(name, "eval", args);
+  *rettv = eval_call_provider(name, "eval", args, 0);
```

Comment:
nit: false/true instead of 0/1.

```suggestion
  *rettv = eval_call_provider(name, "eval", args, false);
```

same for the other cases

Reply:
Done.